pkg/render/manager.go

```diff
@@ -578,12 +578,6 @@ func managerClusterRole(impersonationOnly bool) *rbacv1.ClusterRole {
 
 	if !impersonationOnly {
 		cr.Rules = append(cr.Rules,
-			rbacv1.PolicyRule{
-
-				APIGroups: []string{"authentication.k8s.io"},
-				Resources: []string{"tokenreviews"},
-				Verbs:     []string{"create"},
-			},
```

Comment:
Doesn't es-proxy need a Policyrule for `authenticationreviews`?

Reply:
Please see my comment for the other one above. I tried it out and I think it is not necessary.